### PR TITLE
Alternative travis.yml

### PR DESCRIPTION
**Pull request by Maks3w**
https://gist.github.com/Maks3w/04cfab3312374839529b

### CODE REVIEW

**Comment by weierophinney** [comment]
Can you summarize the differences, please?

**Comment by Maks3w** [comment]
I can't write all differences now. but you can see gist revisions

File: assets/root-files/travis.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
+sudo: false
```
Comment (weierophinney) [inline assets/root-files/travis.yml:1]: Isn't this the default?

File: assets/root-files/travis.yml [hunk 2]
```diff
@@ -8,36 +10,30 @@ php:
...
+install:
+  - if [[ $TRAVIS_PHP_VERSION != '5.6' && $TRAVIS_PHP_VERSION != 'hhvm' && $TRAVIS_PHP_VERSION != 'hhvm-nightly' && $TRAVIS_PHP_VERSION != '7' ]]; then phpenv config-rm xdebug.ini;  fi
+  - composer install --no-interaction --prefer-source
+ 
+before_script:
+  - if [[ $TRAVIS_PHP_VERSION = '5.6' ]]; then mkdir -p build/logs; fi
+ 
+script:
+  - if [[ $TRAVIS_PHP_VERSION = '5.6' ]]; then php ./vendor/bin/phpunit -c tests/phpunit.xml.dist --coverage-clover build/logs/clover.xml; fi
+  - if [[ $TRAVIS_PHP_VERSION != '5.6' ]]; then php ./vendor/bin/phpunit -c tests/phpunit.xml.dist; fi
```
Comment (weierophinney) [inline assets/root-files/travis.yml:34]: As noted in another PR you raised, I want the `phpunit.xml.dist` in the root directory, to simplify running unit tests; if it's present in that directory, we don't need to provided the `-c` flag at all, and it's far simpler for contributors (the majority of projects I've surveyed allow running unit tests from the root directory).

**Comment by weierophinney** [comment]
Looks like the primary differences are:
- Removal of the installation of parallel and libpcre3 (which are no longer necessary)
- Re-ordering of the stanzas (no impact)
- Referencing phpunit.xml.dist under the `test/` tree (which I don't want done)

I'll merge, but will revert the last two items.

File: assets/root-files/travis.yml (hunk 1 above)
Comment (weierophinney) [inline assets/root-files/travis.yml:1]: No, it's not; it's what's needed to opt-in to dockerized builds!
